src/main/java/br/com/faateodoro/nossacasadocodigo/controller/dto/AutorRequest.java

```diff
@@ -0,0 +1,43 @@
+package br.com.faateodoro.nossacasadocodigo.controller.dto;
+
+import br.com.faateodoro.nossacasadocodigo.modelo.Autor;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+public class AutorRequest {
+
+    @NotBlank(message = "O nome é obrigatório")
+    private String nome;
+
+    @Email(message = "O email é obrigatório e deve ter o formato válido")
+    @NotBlank(message = "O email não pode ser vazio")
+    private String email;
+
+    @Size(max = 400, message = "Adescrição nãopode conter mais que 400 caracteres")
```

Comment:
Espaços na mensagem.

```suggestion
    @Size(max = 400, message = "A descrição não pode conter mais que 400 caracteres")
```

Reply:
esse meu teclado tem dificuldades com o espaço =/